catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-left/navigation-left.less

```diff
@@ -52,6 +51,10 @@
         top: @minimumSpacing;
         color: @warning-color;
     }
+
+    img {
+      max-height: 50px;
```

Comment:
This should probably use another unit besides pixels. Any ideas @djblue @andrewkfiedler ?

Reply:
I'm curious what the best unit for this might be as well; I'm not sure either.